lib/Page.js

```diff
@@ -269,13 +269,15 @@ class Page extends EventEmitter {
 
   /**
    * @param {Object} options
+   * @return {!Promise<!ElementHandle>}
```

Comment:
This should probably be `Puppeteer.ElementHandle`, otherwise tsc is not happy.

You can try running tsc locally with
```
npm run tsc
```